scrapy/exporters.py

```diff
@@ -200,7 +200,7 @@ def _build_row(self, values):
             try:
                 yield to_native_str(s)
             except TypeError:
-                yield to_native_str(repr(s))
+                yield to_native_str(str(s))
```

Comment:
a small note: if there is a `str(s)` call there is no need for `to_native_str`, the result will be always the same